src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -27,6 +27,33 @@ public static partial class Path
             (char)31
         };
 
+        //Checks if the given path is available for use.
```

Comment:
style nit -- we put a single space after `//`

Reply:
Resolved